index.html

```diff
@@ -9025,6 +9039,267 @@ <h3><dfn>Take Element Screenshot</dfn></h3>
+
+ <li><p>Let <var>pageWidth</var> be the result of <a>getting a
+  property with default</a> named <code>width</code> and with a
+  default of <code>8.5</code> from <var>page</var>.
+
+ <li><p>Let <var>pageHeight</var> be the result of <a>getting a
+  property with default</a> named <code>width</code> and with a
+  default of <code>11</code> from <var>page</var>.
+
+ <li><p>If either of <var>pageWidth</var> or <var>pageHeight</var> is
+ not an <a>Number</a>, or is less then 0, return <a>error</a>
+ with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>Let <var>margin</var> be the result of <a>getting a property
+  with default</a> named <code>margin</code> and with a default of an
+  empty <a>Object</a> from the <var>parameters</var> argument.
+
+ <li><p>Let <var>marginTop</var> be the result of <a>getting a
+  property with default</a> named <code>top</code> and with a
+  default of <code>0.39</code> from <var>margin</var>.
+
+ <li><p>Let <var>marginBottom</var> be the result of <a>getting a
+  property with default</a> named <code>bottom</code> and with a
+  default of <code>0.39</code> from <var>margin</var>.
+
+ <li><p>Let <var>marginLeft</var> be the result of <a>getting a
+  property with default</a> named <code>left</code> and with a
+  default of <code>0.39</code> from <var>margin</var>.
+
+ <li><p>Let <var>marginRight</var> be the result of <a>getting a
+  property with default</a> named <code>right</code> and with a
+  default of <code>0.39</code> from <var>margin</var>.
+
+ <li><p>If any
+ of <var>martinTop</var>, <var>marginBottom</var>, <var>marginLeft</var>,
+ or <var>marginRight</var> is not an <a>Number</a>, or is less then 0,
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>Let <var>shrinkToFit</var> be the result of <a>getting a
+  property with default</a> named <code>shrinkToFit</code> and with
+  default <code>true</code> from the <var>parameters</var> argument.
+
+ <li><p>If <var>shrinkToFit</var> is not a <a>Boolean</a>
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>Let <var>pageRanges</var> be the result of <a>getting a
+  property with default</a> named <code>pageRanges</code> from
+  the <var>parameters</var> argument with default of an
+  empty <var>Array</var>.
+
+ <li><p>If <var>pageRanges</var> is not an <a>Array</a>
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>When the user agent is next to <a>run the animation frame
+ callbacks</a>, let <var>pdfData</var> be the result of <a>trying</a>
+ to take UA-specific steps to generate a paginated representation of
+ the <a>current browsing context</a> as a PDF, with the following
+ paper settings:
+   <table>
+     <tr>
+       <th>Property
+       <th>Value
+     <tr>
+       <td>Width in inches
+       <td><var>pageWidth</var> if <var>orientation</var> is
+         "<code>portrait</code>" otherwise <var>pageHeight</var>
```

Comment:
I don't think that we have to make a decision based on the `orientation` parameter. The printing engine does it on its own.

Reply:
Right, sure I don't think this would appear in code we write. But I think you need to specify what the effect of the parameter is in some way.